core/src/components/radio-group/radio-group.tsx

```diff
@@ -144,7 +144,9 @@ export class RadioGroup implements ComponentInterface {
       // space bar on top of a selected radio (only applies
       // to radios in a select popover)
```

Comment:
```suggestion
      // space bar on top of a selected radio
```

Remove that extra comment since it is no longer true